tests/test_filter_count.py

```diff
@@ -332,42 +465,110 @@ def test_pruning_order_method():
 @pytest.mark.parametrize(
     "filter_by", ["prune_by_noise_rate", "prune_by_class", "both", "confident_learning"]
 )
-def test_find_label_issues_multi_label(multi_label, filter_by):
+@pytest.mark.parametrize(
+    "return_indices_ranked_by",
+    [None, "self_confidence", "normalized_margin", "confidence_weighted_entropy"],
+)
+def test_find_label_issues_multi_label(multi_label, filter_by, return_indices_ranked_by):
     """Note: argmax_not_equal method is not compatible with multi_label == True"""
+    dataset = multilabel_data if multi_label else data
 
-    s_ml = [[z, data["true_labels_train"][i]] for i, z in enumerate(data["labels"])]
     noise_idx = filter.find_label_issues(
-        labels=s_ml if multi_label else data["labels"],
-        pred_probs=data["pred_probs"],
+        labels=dataset["labels"],
+        pred_probs=dataset["pred_probs"],
         filter_by=filter_by,
         multi_label=multi_label,
+        return_indices_ranked_by=return_indices_ranked_by,
+    )
+    if return_indices_ranked_by is not None:
+        noise_bool = np.zeros(len(dataset["labels"])).astype(bool)
+        noise_bool[noise_idx] = True
+        noise_idx = noise_bool
+    acc = np.mean(
+        (
+            np.array(dataset["labels"], dtype=np.object_)
+            != np.array(dataset["true_labels_train"], dtype=np.object_)
+        )
+        == noise_idx
     )
-    acc = np.mean((data["labels"] != data["true_labels_train"]) == noise_idx)
     # Make sure cleanlab does reasonably well finding the errors.
     # acc is the accuracy of detecting a label error.
     assert acc > 0.85
 
 
+@pytest.mark.parametrize(
+    "confident_joint",
+    [
+        None,
+        [[[1, 0], [0, 4]], [[3, 0], [0, 2]], [[3, 0], [1, 1]], [[3, 1], [0, 1]]],
+        [[1, 1, 0, 2], [0, 1, 0, 1], [0, 0, 1, 1], [0, 0, 0, 1]],
+    ],
+)
+@pytest.mark.parametrize(
+    "return_indices_ranked_by",
+    [None, "self_confidence", "normalized_margin", "confidence_weighted_entropy"],
+)
+@pytest.mark.filterwarnings("ignore::UserWarning")
```

Comment:
what warning are you filtering here?

Reply:
This warning that we print when confident joint is of the old formatting shape


https://github.com/aditya1503/cleanlab/blob/682bd91957f54191add1c892e39714fb6a05af6e/cleanlab/filter.py#L544